src/support.cpp

```diff
@@ -170,6 +170,7 @@ void AreaSupport::generateSupportAreas(SliceDataStorage& storage, unsigned int m
     const int supportMinAreaSqrt = mesh.getSettingInMicrons("support_minimal_diameter");
     const double supportTowerRoofAngle = mesh.getSettingInAngleRadians("support_tower_roof_angle");
     const bool use_towers = mesh.getSettingBoolean("support_use_towers") && supportMinAreaSqrt > 0;
+    const coord_t tower_top_layer_count = 6; // number of layers after which to conclude that a tiny support area needs a tower
```

Comment:
This documentation is incorrect, isn't it? It seems to behave like the support roof height, rather than a simple threshold for tower height of tiny towers.